src/app/DIAmeterApplication.h

```diff
@@ -41,13 +41,25 @@ class DIAmeterApplication : public CruxApplication {
 
  protected:
   // string output_file_name_;
-  double avg_noise_intensity_logrank_;
+  double avg_noise_intensity_logrank_, avg_ms1_peaknum_, avg_ms1_intercept_;
+  int scan_gap_;
+  int max_ms1scan_;
+  int max_ms1_mzbin_, max_ms2_mzbin_;
+
+  std::string remove_index_, output_pin_, output_percolator_;
 
   vector<InputFile> getInputFiles(const vector<string>& filepaths, int ms_level) const;
 
   SpectrumCollection* loadSpectra(const std::string& file);
 
-  double loadMS1Spectra(const std::string& file, map<int, pair<double*, double*>>* ms1scan_intensity_rank_map);
+  // void loadMS1SpectraOld(const std::string& file, map<int, pair<double*, double*>>* ms1scan_intensity_rank_map);
```

Comment:
why is there an "old" and "new"? remove reference to old and new.
Also remove this commented line